lib/siwapp/schema/item.ex

```diff
@@ -9,5 +11,17 @@ defmodule Siwapp.Schema.Item do
     field :unitary_cost, :integer, default: 0
     field :deleted_at, :utc_datetime
     belongs_to :invoice, Invoice
+    many_to_many :taxes, Tax, join_through: "items_taxes"
+  end
+
+  def changeset(item, attrs) do
+    item
+    |> cast(attrs, @fields)
+    |> cast_assoc(:taxes)
+    |> foreign_key_constraint(:invoice_id)
+    |> validate_length(:description, max: 20000)
+    |> validate_number(:quantity, greater_than_or_equal_to: 0)
+    |> validate_number(:discount, greater_than_or_equal_to: 0)
+    |> validate_number(:unitary_cost, greater_than_or_equal_to: 0)
```

Comment:
I'm gonna merge, but be aware that there could be negative costs, so you should allow negative numbers in unitary_cost